lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -727,6 +729,12 @@ monitorStakePools tr (NetworkParameters gp sp _pp) nl DBLayer{..} =
             toChainPoint (BlockHeader  0 _ _ _) = ChainPointAtGenesis
             toChainPoint (BlockHeader sl _ h _) = ChainPoint sl h
 
+
+        let psudoGenesisSlotNo = SlotNo 0
+        atomically $ do
+            putPoolCertificates psudoGenesisSlotNo genesisPools
```

Comment:
This may be sketchy (what happens if you restart the wallet?), but is only used in integration tests, and deserves a notice ⚠️